hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/service/ContractCallService.java

```diff
@@ -190,20 +120,26 @@ private void restoreGasToBucket(HederaEvmTransactionProcessingResult result, lon
         }
     }
 
-    private void validateResult(final HederaEvmTransactionProcessingResult txnResult, final CallType type) {
+    protected void validateResult(final HederaEvmTransactionProcessingResult txnResult, final ContractExecutionParameters.CallType type) {
```

Comment:
nit: use static import for `ContractCallParameters.CallType` and change this to:
```suggestion
    protected void validateResult(final HederaEvmTransactionProcessingResult txnResult, final CallType type) {
```

Same for all other occurrences.